src/main/java/GameEntities/Tiles/JailTile.java

```diff
@@ -26,6 +26,6 @@ public JailTile() {
      */
     @Override
     public TileActionResultModel action(Player player) {
-        return new TileActionResultModel(player, -1);
+        return new TileActionResultModel("You are just visiting jail", -1);
```

Comment:
this flavor text should only be used when the player has landed on jail without actually being in jail